Bio/Pathway/__init__.py

```diff
@@ -104,11 +105,13 @@ def __hash__(self):
 
     def __repr__(self):
         """Return a debugging string representation of self."""
-        return "Reaction(" + \
-               ",".join(map(repr, [self.reactants,
-                                   self.catalysts,
-                                   self.data,
-                                   self.reversible])) + ")"
+        return (
+            "Reaction("
+            + ",".join(
+                map(repr, [self.reactants, self.catalysts, self.data, self.reversible])
+            )
+            + ")"
+        )
```

Comment:
This isn't a black issue, but using map seems to have gone out of fashion with list comprehensions being added to Python. However, I would just write this as follows - line wrapped as needed:

```python
        return "Reaction(%r, %r, %r, %r)" % (self.reactants, self.catalysts, self.reversible, self.data)
```

As an aside, note that I have corrected the argument order to match ``__init__``, since the convention is that ``object == eval(repr(object))`` for simple Python objects.

Reply:
Thank you, I wouldn't had pick that.